src/commands/subdomain.rs

```diff
@@ -82,44 +65,24 @@ fn register_subdomain(
     message::working(&msg);
     let account_id = &target.account_id;
     let addr = subdomain_addr(account_id);
-    let sd = Subdomain {
+    let subdomain = Subdomain {
         subdomain: name.to_string(),
     };
-    let sd_request = serde_json::to_string(&sd)?;
+    let subdomain_request = serde_json::to_string(&subdomain)?;
 
     let client = http::auth_client(None, user);
 
-    let mut res = client.put(&addr).body(sd_request).send()?;
+    let mut res = client.put(&addr).body(subdomain_request).send()?;
```

Comment:
a last tiny nit... this would be awesome to have as a Subdomain::put method! What do you think?